redwood-schema/src/main/kotlin/app/cash/redwood/schema/parser/schemaParser.kt

```diff
@@ -95,18 +95,24 @@ public fun parseSchema(schemaType: KClass<*>, tag: UInt = 0U): Schema {
     throw IllegalArgumentException(
       buildString {
         appendLine("Schema @LayoutModifier tags must be unique")
-        for ((tag, group) in badLayoutModifiers) {
-          append("\n- @LayoutModifier($tag): ")
+        for ((modifierTag, group) in badLayoutModifiers) {
+          append("\n- @LayoutModifier($modifierTag): ")
           group.joinTo(this) { it.type.qualifiedName!! }
         }
       },
     )
   }
 
-  val scopes = widgets
+  val widgetScopes = widgets
     .flatMap { it.traits }
     .filterIsInstance<Widget.Children>()
-    .mapNotNullTo(mutableSetOf()) { it.scope }
+    .mapNotNull { it.scope }
+  val layoutModifierScopes = layoutModifiers
+    .flatMap { it.scopes }
+  val scopes = buildSet {
+    addAll(widgetScopes)
+    addAll(layoutModifierScopes)
+  }
```

Comment:
This fixes a minor issue where scopes aren't created if they're only referenced by a layout modifier and are not referenced by any widgets.